### PR TITLE
Input Validation for Keys

### PR DESCRIPTION
**Pull request by vcidst**
Current API would crash if it sees a KeyError, this commit fixes that

### CODE REVIEW

**Comment by NirantK** [comment]
Requested review from @meghanabhange who is the author+maintainer of that API.

File: relationship_server.py [hunk 1]
```diff
@@ -132,9 +132,18 @@ def process_records(records):
 
 @app.route("/", methods=["POST"])
 def single():
-    req_data = request.get_json()
+    try:
+        req_data = request.get_json()
+        results = process_records(req_data)
+    except TypeError:
+        # abort when not JSON
+        abort(400)
```
Comment (meghanabhange) [inline relationship_server.py:139]: There are import errors with abort and jsonify. 
Please import `from flask import jsonify, abort`

Comment (vcidst) [reply relationship_server.py:139]: I'm sorry there were uncommitted changes in my repo that I forgot about!

**Comment by vcidst** [comment]
Sorry about the ImportError! Fixed in 4374f8600346c6bf2ebfa0f28363f5a45cac2730

**Pull request merged**